build.sh

```diff
@@ -23,12 +23,21 @@ elif [[ "$image" == *-xenial* ]]; then
   UBUNTU_VERSION=16.04
 elif [[ "$image" == *-artful* ]]; then
   UBUNTU_VERSION=17.10
+elif [[ "$image" == *-bionic* ]]; then
+  echo "found version "
+  UBUNTU_VERSION=18.04
 fi
 
 # It's annoying to rename jobs every time you want to rewrite a
 # configuration, so we hardcode everything here rather than do it
 # from scratch
 case "$image" in
+  pytorch-linux-bionic-clang-9)
+    CLANG_VERSION=9
+    THRIFT=yes
```

Comment:
Since none of the other containers have thrift in them, shouldn't we also say that this image contains thrift?

Reply:
(in the environment name)

Reply:
do you mean something like: pytorch-linux-bionic-clang-9-thrift?

Reply:
yes